db/emoji.json

```diff
@@ -15306,6 +15306,7 @@
   , "category": "Objects"
   , "aliases": [
       "e-mail"
+    , "email"
```

Comment:
Maybe the `email` alias could be canonical (i.e. first) and `e-mail` added as an additional alias? That way we could let `e-mail` be slowly phased out over time, as it will no longer expand to the dash version from various emoji pickers built on this library.

Reply:
That's a good idea. I had not thought of it that way. I have made this change in ce6c4ab12ae229be2b1089cbf7e85702fdc5552f.

Reply:
I have a question my ex put all these codes and things in my phone how do I get them out.